### ISSUE TITLE
Make Spectrum->Document conversion part of Spec2Vec class

### ISSUE THREAD

**Issue by florian-huber** [iomega/spec2vec#51]
inspired by recent work we did on `MS2DeepScore` -> https://github.com/matchms/ms2deepscore/issues/23

- Define a Spec2VecModel that contains the `Word2Vec` gensim model as well as a the parameters used to create the `SpectrumDocument`
- Incorporate the `Spectrum` to `SpectrumDocument` conversion into the scoring function, this would allow to have a similarity score that takes `Spectrum` as input (same as the `matchms` similarity functions) --> Check performance loss!

**Comment by florian-huber** [comment]
For now I skipped making an own `Spec2VecModel` class, but sticked to the normal gensim `Word2Vec` model.
But I implemented a method to get the decimal rounding from the model vocabulary to `Spec2Vec` ( #54). I also incorporated the `Spectrum` to `SpectrumDocument` conversion (if necessary, that is if the input is of type `Spectrum`). This way the workflow will become easier for the user, in particular when combining multiple scores (Cosine, mod.Cosine, Spec2Vec, MS2DeepScore).
